tests/pkg_libfixmath/main.c

```diff
@@ -59,8 +60,20 @@ static void binary_ops(void)
         { "max", fix16_max },
     };
 
-    for (fix16_t a = fix16_from_dbl(-5.0); a < fix16_from_dbl(5.0); a += fix16_from_dbl(0.25)) {
-        for (fix16_t b = fix16_from_dbl(-5.0); b < fix16_from_dbl(5.0); b += fix16_from_dbl(0.25)) {
+#ifdef BOARD_NATIVE
+    fix16_t _min = fix16_from_dbl(-5.0);
+    fix16_t _max = fix16_from_dbl(5.0);
+    fix16_t _step = fix16_from_dbl(0.25);
+#else
+    fix16_t _min = fix16_from_dbl(-2.0);
+    fix16_t _max = fix16_from_dbl(2.0);
+    fix16_t _step = fix16_from_dbl(0.25);
+#endif
```

Comment:
The new `IS_USED()` feature could be used here. (But I'm not insisting.)

```C
const double _range = (IS_USED(BOARD_NATIVE)) ? 5.0 : 2.0;
fix16_t _min = fix16_from_dbl(- _range);
fix16_t _max = fix16_from_dbl(_range);
fix16_t _step = fix16_from_dbl(0.25);
```

Reply:
Indeed. But for macros directly related to the target board, I think `IS_USED()` is not really helpful since the build on Murdock will cover all the cases.